src/uk/gov/hmcts/contino/azure/Acr.groovy

```diff
@@ -52,14 +52,20 @@ class Acr extends Az {
   /**
    * Build an image
    *
+   * Notice that when an image is tagged as 'latest', another tag
+   * with a suffix appended 'latest' is also created to trigger
+   * possible downstream deployments:
+   * e.g. <image-name>:latest will also be tagged <image-name>:latest_dfb02
+   *
    * @param dockerImage
    *   the docker image to build
    *
    * @return
    *   stdout of the step
    */
   def build(DockerImage dockerImage) {
-    this.az "acr build --no-format -r ${registryName} -t ${dockerImage.getShortName()} -g ${resourceGroup} ."
+    def additionalTag = (dockerImage.getTag() != "latest") ? "" : "-t ${dockerImage.getShortName()}_{{.Run.ID}} "
```

Comment:
nitpicking here, but maybe `-` instead of `_` is more conventional?

Reply:
OK sure np